README.md

```diff
@@ -1,11 +1,6 @@
 # Slack Data Collector
 
-    version: alpha-0.0.1
-
-| Branch | Build Status |
-| ------ | --- |
-| master | [![Build Status](https://travis-ci.org/mena-devs/slack_data_collector.svg?branch=master)](https://travis-ci.org/mena-devs/slack_data_collector) |
-| alpha-0.0.2 | [![Build Status](https://travis-ci.org/mena-devs/slack_data_collector.svg?branch=alpha-0.0.2)](https://travis-ci.org/mena-devs/slack_data_collector) |
```

Comment:
This one, we keep :) It's good to know the status of the development branch in case someone wants to get the latest.